notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,484 @@
+{
```

Comment:
<p>Typo: "PClass" should be "Pclass"</p><p> </p><p>Also, I think one useful point to mention here is that many pandas operations return a DataFrame or a Series, and this is an example of a pandas operation (describe) returning a Series.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>